pkg/scheduler/framework/types.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
the suggestion was actually to do the opposite: to have this file in `scheduler/framework/v1alpha1` and the original one moved here (to `scheduler/framework`), here we only need the types that external components like kubelet needs.